packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md

```diff
@@ -0,0 +1,163 @@
+
+<div class="full border">
+
+![An illustration that shows the significant complexity of a full IAM implementation.](/img/concepts/IAM/03-architecture-example.png)
+
+</div>
+
+
+The figure shows a few aspects of the IAM solution:
+
+- Cloud-based IAM platform
+
+- AWS compatibility
+
+- Single Sign-On (SSO)
+
+- External Identity Providers
+
+- Social authentication
+
+- System for Cross-domain Identity Management (SCIM) provisioning
+
+- OAuth 2.0 and OpenID Connect (OIDC) authorization and authentication
+
+- Security Assertion Markup Language (SAML) authentication
+
+- Multi-Factor Authentication (MFA) and Universal 2nd Factor (U2F) authentication
+
+- User-specific authentication flows
+
+- HIPAA compliance
+
+- Mobile and desktop device support
+
+- Self-registration
+
+- IAM security logging
+
+- B2B and B2C support
+
+- Customer administration
+
+Architectural properties such as reliability, availability, performance, scalability, and ease of integration; functions such as sign-in, access control, and administration; and customization such as branding and tailored flows, are difficult to illustrate, but were among the organization's many considerations in the design of its IAM solution.
+
+## Designing an IAM solution
+
+The design of any IAM solution must consider the following areas:
+
+- **Identity management:** Store and manage data to uniquely identify every authorized individual, business, device, app, and other resource, along with their attributes and policies. This is your main source of data for user authentication, authorization, and access control.
+
+- **Authentication:** Verify that user login credentials are both legitimate and being used by their owners.
+
+- **Authorization:** Define what resources a given user is allowed to access and what functions they are allowed to perform with them.
+
+- **Access control:** Grant or deny individual requests to view or update a restricted resource based on the resource, the nature of the request, whether the user is authenticated, the user's authorizations, relevant policies, and other data. (Access control is part of authorization.)
+
+All of these functions must be highly reliable, available, secure, and performant.
+
+> **Note:** The figure in [Features of an IAM solution](#features-of-an-iam-solution) compactly summarizes important features to consider, but some of them fall into two or more of the functional areas above. For more clarity, features in this section are grouped by the main area in which they apply. The areas are ordered as above, with access control included with authorization.
+
+Subsequent articles in this series drill down on the main issues, factors, and strategies to consider for each area above, along with how Okta can help reduce your time, effort, costs, and risks for developing, fielding, maintaining, and supporting an IAM solution that fits seamlessly with your applications and is tailored to your needs. The articles also include
```

Comment:
This may be too much, but given the stated goals of the documents, it fits:

> Other articles in this series give more detail on the issues, factors, and strategies for specific areas.